src/StudyUtils.in.jsm

```diff
@@ -197,24 +323,48 @@ class StudyUtils {
     this._isSetup = true;
     return this;
   }
+
+  /*
+  * TODO bdanforth: add docblock'
+  * @QUESTION: When would this be called? It's not called at all in Kamyar's study. At bootstrap shutdown?
+  */
   reset() {
     this.config = {};
     delete this._variation;
     this._isSetup = false;
   }
+  /**
+  * @async
+  * TODO bdanforth: add docblock
+  * params info: http://mdn.beonex.com/en/XUL/Method/addTab.html
+  */
   async openTab(url, params = {}) {
     this.throwIfNotSetup("openTab");
     log.debug(url, params);
+    /*
+    * @QUESTION: Why are we waiting 30 seconds to add a tab?
+    */
     log.debug("opening this formatted tab", url, params);
     if (!Services.wm.getMostRecentWindow("navigator:browser").gBrowser) {
       // Wait for the window to be opened
       await new Promise(resolve => setTimeout(resolve, 30000));
     }
     Services.wm.getMostRecentWindow("navigator:browser").gBrowser.addTab(url, params);
   }
+
+  /**
+  * @async
+  * TODO bdanforth: add docblock
+  * @QUESTIONS: Why is there a ShieldUtils method and a module function for `getTelemetryId`?
```

Comment:
No good reason.

Reply:
(maybe used by smapling stuff which is outside the class?)

Reply:
I did a ctrl+F and I don't see it called anywhere except inside StudyUtils methods. Okay to move it in?